packages/core/src/components/Review/ReviewLink.jsx

```diff
@@ -0,0 +1,41 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export class ReviewLink extends React.PureComponent {
+  handleClick(event) {
+    if (this.props.onClick) {
+      this.props.onClick(event, this.props.href);
+    }
+  }
+
+  render() {
+    const { href, className, children, ariaLabel } = this.props;
+    const onClick = event => this.handleClick(event);
+    return (
+      <div className="ds-u-margin--0">
```

Comment:
Is this margin necessary? Disabling it doesn't seem to have an affect on anything from what I can tell.